plugins/admin/go.mod

```diff
@@ -7,7 +7,8 @@ require (
 	github.com/spf13/cobra v0.0.5
 	github.com/spf13/viper v1.6.2
 	gotest.tools v2.2.0+incompatible
-	k8s.io/api v0.17.3
-	k8s.io/apimachinery v0.17.3
-	k8s.io/client-go v0.17.0
+	k8s.io/api v0.17.4
+	k8s.io/apimachinery v0.17.4
+	k8s.io/client-go v0.17.4
+	knative.dev/client v0.14.0
```

Comment:
why do you need a dependency on `knative.dev/client` ?

Reply:
@rhuss
I noticed there's a method in knative.dev/client to create flags like --scale-to-zero and --no-scale-to-zero, so reused it.

```
flags.AddBothBoolFlagsUnhidden
```

Reply:
I think this is ok for now but for the long run we should extract the parts that can be reused by plugins within the client into at least a dedicated package. Because the client coders *will not be aware* that this method is used by anybody outside the client itself, so its very likely that this can be changed by a refactoring or by accident.